livereload.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+const { spawn } = require("child_process");
+const { normalize } = require("path");
+
+const termiateSignals = ["close", "exit", ""];

```

Comment:
Could you please reword this to `terminationSignals`.

Could you explain when we might receive a "" signal?

Should we also handle a "[disconnect](https://nodejs.org/api/child_process.html#child_process_event_disconnect)" signal?

Reply:
The "" signal, was just a leftover as I first added the "disconnect" event. Then I figured out we do not need it, as it only happens, when disconnect is explicitly called. The renaming of the variable makes sense and is done,